### ISSUE TITLE
Comma incorrectly inserted after family name

### ISSUE THREAD

**Issue by DilumAluthge** [marcodaniel/biblatex-nejm#4]
# Description of Issue
The current version of `biblatex-nejm` inserts a comma after the family name (last name) of each author. However, the [examples of formatted references](http://www.nejm.org/page/author-center/manuscript-submission#style) on the NEJM website do not include commas after the family name.

# Minimum Working Example (MWE)
Step 1: Put the following into a file called `example.tex`:
```tex
\documentclass{article}

\usepackage[backend=biber, style=nejm]{biblatex}
\addbibresource{mybibfile.bib}

\begin{document}

First citation: \cite{Kuczmarski2000}. Second citation: \cite{usposition2002}. Third citation: \cite{Goadsby2001}. Fourth citation: \cite{Shapiro2000}.

\printbibliography

\end{document}
```
Step 2: Put the following into `mybibfile.bib`:
```
%% Examples for biblatex-nejm
@ARTICLE{Shapiro2000,
author   = "A.M. James Shapiro and Jonathan R.T. Lakey and
Edmond A. Ryan and Gregory S. Korbutt and Ellen Toth and
Garth L. Warnock and Norman M. Kneteman and Ray V. Rajotte",
title    = "Islet transplantation in seven patients with type 1 diabetes
mellitus using a glucocorticoid-free immunosuppressive regimen",
journal  = "N Engl J Med",
year     = "2000",
number   = "2",
volume   = "343",
pages    = "230-8"
}
@ARTICLE{Goadsby2001,
author   = "Peter J. Goadsby",
title    = "Pathophysiology of headache",
journaltitle = "Silberstein SD, Lipton RB, D'Alessio DJ, eds",
journalsubtitle="Wolff's Headache and Other Head Pain",
location = " {Oxford, England}",
publisher= "Oxford University Press",
edition  = "7",
year     = "2001",
pages    = "57-72"
}
@ONLINE{usposition2002,
title    = "U.S. positions on selected issues at the third negotiating
session of the Framework Convention on Tobacco Control",
location = " {Washington, D.C.}",
publisher= "Committee on Government Reform",
year     = "2002",
addendum = "(Accessed March 4, 2002,
\url{http://www.house.gov/reform/min/inves_tobacco/index_accord.htm})"
}
@ARTICLE{Kuczmarski2000,
author   = "Cynthia L. Ogden and Robert J. Kuczmarski and Katherine M. Flegal
and Zuguo Mei, MD and Shumei Guo and Rong Wei and
Laurence M. Grummer-Strawn and Lester R. Curtin and
Alex F. Roche and Clifford L. Johnson",
title    = "CDC growth charts: United States. Advance data from
vital and health statistics. No. 314",
journaltitle  = "National Center for Health Statistics",
location = " {Hyattsville, Md} ",
year     = "2000",
addendum = "(DHHS publication no. (PHS) 2000-1250 0-0431)"
}
```

Step 3: Run the following commands:
```bash
pdflatex example
biber example
pdflatex example
pdflatex example
```

Step 4: Open the file `example.pdf` in your favorite PDF viewer. This is the result:
![screen shot 2017-09-26 at 9 42 16 pm](https://user-images.githubusercontent.com/5619885/30892008-3fe19fae-a304-11e7-9efa-423175f5ef7d.png)

**Comment by dilumaluthge** [comment]
Pull request #5 fixes this issue. Here is a screenshot of the previous MWE after PR #5 is applied:

![screen shot 2017-09-26 at 9 48 22 pm](https://user-images.githubusercontent.com/5619885/30892457-ebfd3940-a306-11e7-9d66-45e44e473c72.png)

**Comment by DilumAluthge** [comment]
Closed by #5
